Hooks/FluentCrmHooks.php

```
@@ -1,117 +1,146 @@
+			}
+		);
+	}
+
+	/**
+	 * Shape a stored conversation row into a FluentCRM ticket entry.
+	 *
+	 * @param object $conversation Row from the conversations table.
+	 * @return array
+	 */
+	public static function format_ticket( object $conversation ): array {
+		$conversation_url = THRIVEDESK_APP_URL . '/conversations/' . $conversation->id;
```

PHPStan (2.2.5)

[error] Constant THRIVEDESK_APP_URL not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)